app/api/emojis/index.js

```diff
@@ -0,0 +1,104 @@
+const express = require('express');
+const Emojis = require('../../database/models/emojiModel');
+const Users = require('../../database/models/usersModel');
+
+const logger = require('../../logger');
+
+const checkFields = require('../../../util/checkFields');
+
+const router = express.Router();
+
+// Get Emojis by post id
+router.put('/', async (req, res) => {
```

Comment:
Yep was a quick hack cause GET requests dont get a body

Reply:
Yeah, we should put the fields in the query parameters in this case. There are practical reasons for this too, not just ideological church-of-REST reasons :). Since GETs are idempotent most clients can do auto-retries, caching, and things like that.